### ISSUE TITLE
[Bug Report] text-truncate cuts bottom of text

### ISSUE THREAD

**Issue by joaobarcia** [vuetifyjs/vuetify#12032]
### Environment
**Vuetify Version:** 2.3.8
**Vue Version:** 2.6.11
**Browsers:** Chrome 84.0.4147.105
**OS:** Windows 10

### Steps to reproduce
Create a div with classes "title" and "text-truncate"

```
<div class=title text-truncate">
gggyyy
</div>
```

### Expected Behavior
Bottom of text should not be cut off:

### Actual Behavior
Bottom of text is being cut off 
![Cropped Text](https://user-images.githubusercontent.com/2707930/52716311-c7817880-2f9e-11e9-81f5-7d9b6a0ae3ba.png)

### Reproduction Link
<a href="https://codepen.io/wistudent/pen/xMJPQq" target="_blank">https://codepen.io/wistudent/pen/xMJPQq</a>

### Other comments
Why not just set `overflow-x: hidden` but leave overflow-y untouched? Is there anything at all to be gained from changing the overflow-y? As far as I understand truncation is only horizontal. Changing it vertically causes unnecessary repercussions.

(Shamelessly re-used content from https://github.com/vuetifyjs/vuetify/issues/6488 since a solution was not provided. Only a workaround which introduces CSS pollution)

**Comment by MajesticPotatoe** [comment]
Seems the fix didn't get carried over from 1.5 styl -> sass conversion. Though does seem more like removing overflow would be the better fix, I cant say why this added (could have been to fix another issue)

**Comment by joaobarcia** [comment]
Thanks for the quick reply @MajesticPotatoe ! Great username 😄 

As a note, in your case we went with a 
```
.text-truncate
  line-height: unset !important
```

Changing the overflows actually generates some issues with scrollbars showing up in the elements.
